src/index.tsx

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './App';
-import { FluentCustomizations } from '@uifabric/fluent-theme';
-import { Customizer, mergeStyles } from 'office-ui-fabric-react';
+import { Customizer, mergeStyles } from '@fluentui/react';
 import * as serviceWorker from './serviceWorker';
 
 // Inject some global styles
 mergeStyles({
-  selectors: {
-    ':global(body), :global(html), :global(#root)': {
-      margin: 0,
-      padding: 0,
-      height: '100vh'
-    }
+  ':global(body,html,#root)': {
+    margin: 0,
+    padding: 0,
+    height: '100vh'
   }
 });
 
 ReactDOM.render(
-  <Customizer {...FluentCustomizations}>
+  <Customizer>
```

Comment:
You should be able to just get rid of the Customizer wrapper now.

Reply:
I see. it wasn't clear from the docs. thx!